.github/workflows/ci.yml

```diff
@@ -293,4 +293,4 @@ jobs:
           docker-username: ${{ secrets.DOCKERHUB_USERNAME }}
           docker-password: ${{ secrets.DOCKERHUB_TOKEN }}
           docker-images: >-
-            springcloudtask/timestamp-batch-task:2.0.0
\ No newline at end of file
+            springcloudtask/timestamp-batch-task:2.0.1
```

Comment:
Do you mind adding a newline here (it was already missing)

Reply:
done